src/main/java/net/minecraftforge/fmp/FMLForgeMultipartPlugin.java

```diff
@@ -0,0 +1,37 @@
+package net.minecraftforge.fmp;
```

Comment:
Not needed. We can do this a bit cleaner, and you don't need to inject on the coremod side.